lib/actions/action.js

```diff
@@ -47,6 +47,20 @@ class Action extends EventAware {
     }
     this.log.info(JSON.stringify(usageLog))
   }
+
+  /**
+   * Returns the actionable issues and/or pulls if it's for a schedule event. Otherwise return the current issue or pull that is for the current event.
+   */
+  getActionables (context, results) {
+    let scheduleResults = (context.event === 'schedule')
+      ? results && results.validationSuites && results.validationSuites.length && results.validationSuites[0].schedule
+      : null
```

Comment:
Should it return an empty array instead of null here so that actionables.map doesn't fail in case nothing is present?